client/webserver/site/src/html/forms.tmpl

```diff
@@ -140,25 +140,48 @@
 {{end}}
 
 {{define "confirmRegistrationForm"}}
-{{$passwordIsCached := .UserInfo.PasswordIsCached}}
-<div class="bg2 px-2 py-1 text-center position-relative fs18">
+<div class="bg2 px-2 py-1 text-center fs18">
   Confirm Registration
   <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
 </div>
 <div class="p-4">
   <div class="fs16">
-    <span {{if $passwordIsCached}}class="d-hide"{{end}}>Enter your app password to confirm DEX registration.</span>
-    When you submit this form, <span id="feeDisplay"></span> DCR will be spent from your Decred wallet to pay
-    registration fees.
+    Enter your app password to confirm DEX registration.
```

Comment:
Seems like this line needs to be hidden conditionally too.

Also, apologies in advance because we're likely going to merge https://github.com/decred/dcrdex/pull/1051 first since we keep making @buck54321 update it.  When that goes in, you'll have to resolve the differences with the `confirmRegistrationForm` template (and any other conflicts), define any new strings in client/webserver/locales/en-us.go, then do the `go generate` command in `client/webserver/site` to rebuild the localized_html files.  This is going to be part of the site workflow going forward, so we might as well trial it with you after #1051 goes in.

Reply:
All good, no problem.